### DISCUSSION TITLE
can not access the application from other pc from the same network

### DISCUSSION THREAD

**Discussion by christorpher** [Budibase/budibase#9044] [Help] [answered]
i have installed budibase but cannot access the application from same network
with 
localhost or same domain ip 
or internal docker host
please hlep

**Comment by shogunpurple** [comment, marked as answer]
Hi @christorpher - just to be clear - are you running budibase on another server on your local network? Or is it hosted on the machine you are currently using?

**Reply by christorpher** [reply]
another sever on the local network  (machine A as server machine B as pc on which we are trying to run the url and also tried other machine no restriction policy)

thank you for trying to resolve my issue
means alot

**Reply by shogunpurple** [reply]
are you using the local IP address of the other server? From your question above, it seems like you are trying to use:
- `localhost` - your local machine, which won't work if the server is another machine
- `same domain ip` - I'm not sure what this means
- `internal docker host` - this will only work on the same machine

You need to get the local IP address of the other server - then in your browser you can type:
`<local-ip>:10000` assuming you have the default self hosted budibase running on the other server, and there's no firewall rules set up to prevent access to that port.

**Reply by christorpher** [reply]
sir i m using <local-ip>:10000 which is 192.xxx.1.250:10000 form another 192.xxx.1.48 by 192.xxx.1.250:10000 keeping this in url have installed bb before but i dont why my other machine arent able to access now
please help

**Reply by shogunpurple** [reply]
If Budibase is accessible on the host machine, this seems like a networking issue within your network - can you access the server machine directly from your own machine? Ie accessing another service, or SSH into it? Just trying to understand if you have any connectivity from your local machine to the server at all, or it’s just a BB problem specifically.

**Reply by christorpher** [reply]
sir i can access the pc from vnc and even there is one more host installed of quick heal (seqrite) which i can access from other terminal from the same network these ports are distinct

**Comment by christorpher** [comment]
THANK YOU FOR THE SUPPORT MY FREIND SOLVED THE PROBLEM
GLAD TO HAVE SUCH PEOPLE AROUND THANKS
THANKS TO BUDIBASE TEAM AND MY VIJAY SIR
